quotes.js

```diff
@@ -59,4 +59,10 @@ module.exports = [
     "People who buy things are suckers.",
     "I'm going to get 12 eggs and part of a dead animal. Dealer's choice. Please and thank you.",
     "I wanna punch you in the face so bad right now."
```

Comment:
You need to insert a comma at the end of line 61. This project was first deployed before Node.js supported trailing commas.

Reply:
removed trailing comma at the last array entry

Reply:
and added comma after line 61 😄

Reply:
Amazing! I'll get this merged!